Issue 7: Volume Control

joshuali7536 opened the pull request:
Fixes #7. I have added a volume control slider to the top nav bar that will adjust the playback volume. It also has been adjusted to match the theme of the application. Please let me know if there is anything you would like adjusted.

I implemented this by:
- added new html and css for the slider
- adding a new `volumeChange()` function that updates the `audioPreviewPlayer.volume` property
- modifying `preview()` to adjust the volume property when creating a new `Audio` object

Preview of changes:
![image](https://user-images.githubusercontent.com/49960519/139562949-1c9976c7-5907-4af4-87c6-b60757e20ef6.png)

stxarixdust commented:
This looks very nice! Two possible enhancements: 
- Could you replace the "Volume" label with one of the feather icons? I believe there is a speaker icon in the set.
- The top bar looks a little cramped with the volume slider aligned to the left. Could you align it to the right instead?

joshuali7536 commented:
I have adjusted the volume elements to be right aligned and modified the label to use the feather volume icons instead. They also responsively change when the volume is at different levels.

![Demo](https://i.gyazo.com/d0be72881b3db6955b278661bcc38dee.gif)

I also added a mute toggle when you click on the speaker icon. Let me know if there's anything else you want changed!

stxarixdust commented:
Nice additions! Looks all good and ready to merge.
